core/src/avm2/globals/flash/utils.rs

```diff
@@ -25,6 +25,66 @@ pub fn get_timer<'gc>(
         .into())
 }
 
+/// Implements `flash.utils.setInterval`
+pub fn set_interval<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    let (args, params) = args.split_at(2);
```

Comment:
This panics if there are less than 2 arguments, lets check the length and return an error if there are less than 2 before doing this